app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    source_path = parts[1]
```

Comment:
You need to validate that command have exactly 3 arguments

Reply:
This is not fixed